cloud/gcp/terraform.tfvars.template

```diff
@@ -1,15 +1,15 @@
 # GCP Settings
-gcp_project_id   = "GCP-PROJECT-ID" # CHANGE ME
-gcp_project_name   = "GCP-PROJECT-NAME" # CHANGE ME
-gcp_region    = "us-central1" # CHANGE ME
+gcp_project_id   = "@GCP_PROJECT_ID@"
+gcp_project_name   = "@GCP_PROJECT_NAME@"
+gcp_region    = "@GCP_PROJECT_REGION@"
 gcp_auth_file = "./auth/credentials.json"
 
 # Storage Buckets
-tf-state-bucket-name   = "udmi-terraform-state-bucket"
+tf-state-bucket-name   = "@GCP_PROJECT_NAME@-terraform"
 tf-state-storage-class = "STANDARD"
 
 # Access
-udmi_access_group = "group:group@example.com" # CHANGE ME
+udmi_access_group = "group:@UDMI_SITE_GROUP@"
```

Comment:
This group account is to give admin access to the resources, so it should have a different variable than the one in the udmi-sites.tf file

Reply:
done